pep8speaks/helpers.py

```diff
@@ -189,11 +166,9 @@ def run_pycodestyle(data, config):
 
     # Run pycodestyle
     r = requests.get(diff_url, headers=diff_headers, auth=auth)
-    with open(".diff", "w+") as diff_file:
-        diff_file.write(r.text)
 
     ## All the python files with additions
-    patch = unidiff.PatchSet.from_filename('.diff', encoding='utf-8')
+    patch = unidiff.PatchSet(r.text.splitlines(), encoding='utf-8')
```

Comment:
I have changed it to split on `.content`(bytes) rather than `.text`, and we will pass the encoding used by `requests` to `PatchSet` method.